dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackerOwnershipManagerTest.java

```diff
@@ -426,6 +443,21 @@ void shouldFindTrackedEntityWhenTransferredToAccessibleOrgUnit()
         trackedEntities.stream().map(BaseIdentifiableObject::getUid).toList());
   }
 
+  @Test
+  void shouldFindTrackedEntityWhenTransferredToAccessibleOrgUnitAndSuperUser()
+      throws ForbiddenException, BadRequestException, NotFoundException {
+    transferOwnership(trackedEntityA1, programA, organisationUnitB);
+    superUser.setOrganisationUnits(Set.of(organisationUnitB));
+    userService.updateUser(superUser);
+    TrackedEntityOperationParams operationParams = createOperationParams(superUser, null);
+
+    List<TrackedEntity> trackedEntities = trackedEntityService.getTrackedEntities(operationParams);
+
+    assertContainsOnly(
```

Comment:
We should adopt some of the practices from other tests like https://github.com/dhis2/dhis2-core/blob/8cf7036ab8ef79b8830071d289d77eb271de9355/dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java#L354-L357 to make them as readable.